lib/pages/discover.dart

```diff
@@ -24,7 +24,7 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
 
     final stream = NetworkAnalyzer.discover2(subnet, port);
 
-    deviceList = [];
+    deviceList = ['192.168.1.1'];
```

Comment:
You forgot to remove the test element.

Reply:
@sanket143 yeah sorry about that one :sweat_smile: removed it